ddtrace/ext/app_types.go

```diff
@@ -52,4 +52,9 @@ const (
 	// SpanTypeElasticSearch marks a span as an ElasticSearch operation.
 	// These spans may also have an "elasticsearch.body" tag.
 	SpanTypeElasticSearch = "elasticsearch"
+
+	// SpanTypeMessageConsumer marks a span as a queue operation
+	SpanTypeMessageConsumer = "queue"
```

Comment:
Should we differentiate between these two? @palazzem any suggestions? Other instrumentations do.